src/__tests__/accessiblity/accessibility.test.tsx

```diff
@@ -73,13 +73,13 @@ describe('Accessibility', () => {
 
   // TO DO: resolve 'Axe is already running' issue and re-enable test
   // https://github.com/mozilla/perfcompare/issues/222
-  // it('CompareResultsView should have no violations in dark mode', async () => {
-  //   const { testData } = getTestData();
-  //   const selectedRevisions = testData.slice(0, 4);
-  //   store.dispatch(setSelectedRevisions(selectedRevisions));
+  it('CompareResultsView should have no violations in dark mode', async () => {
+    const { testData } = getTestData();
+    const selectedRevisions = testData.slice(0, 4);
+    store.dispatch(setSelectedRevisions(selectedRevisions));
 
-  //   const { container } = renderWithRouter(<CompareResultsView mode="dark" />);
-  //   const results = await axe(container);
-  //   expect(results).toHaveNoViolations();
-  // });
+    const { container } = renderWithRouter(<CompareResultsView mode="dark" />);
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
```

Comment:
Why did you re-enable this test?

Reply:
Hello @jrbenny35.

I had to re-enable (uncomment) the test because after I ran the Jest command: `npm run test -- -u` an issue occurred in [this file](https://github.com/mozilla/perfcompare/pull/369/files/57a97f827687316f978f298afeb2e54667d9e6c3#diff-bc254208bac8f0db6c81898769662811a2bfd113cfddbeecc26f4c5ccfbf5fa0). The log in the terminal indicated the issue was on **lines 81 to 84.**

In a bid to resolve the issue, I uncommented the lines. And when I then run `npm run test -- -u` command again, the test passed successfully.

Kindly note that I had actually thought I was the one who commented out the block of code in question. I thought I did so myself at some point earlier during development.